spec/models/green_flag/feature_spec.rb

```diff
@@ -132,4 +132,53 @@
       end
     end
   end
+
+  describe '#requires_manual_deletion?' do
+    let(:feature)           { GreenFlag::Feature.create(code: 'big_font') }
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.